libsolidity/codegen/CompilerContext.cpp

```diff
@@ -455,7 +455,9 @@ void CompilerContext::appendInlineAssembly(
 			_assembly + "\n"
 			"------------------ Errors: ----------------\n";
 		for (auto const& error: errorReporter.errors())
-			message += SourceReferenceFormatter::formatErrorInformation(*error);
+			// TODO if we have "locationOverride", it will be the wrong scanner,
```

Comment:
todo

Reply:
Cannot fix this now, it is also not really worse. We will have to see how this looks with all the changes combined. Also, this is just an error that is reported in case the code generator generates invalid yul. It should not be a problem with invalid inline assembly.